backend/src/nodes/impl/color/convert_model.py

```diff
@@ -10,15 +10,15 @@
 
 class ColorSpace:
     def __init__(self, id_: int, name: str, channels: int):
-        assert 0 <= id_ and id_ < 256
+        assert id_ >= 0 and id_ < 256
```

Comment:
This is one of those yoda rules, right? Could we disable that one? `0 <= x && x <= 100` and similar is a common pattern to check whether a value is within a range. This code change just makes that pattern harder to recognize.

And no, I don't like the `0 <= x <= 100` python way, because it's python specific and requires that you know python's rules for auto inserting `and`.